astroquery/jplhorizons/core.py

```diff
@@ -1181,14 +1214,21 @@ def _parse_result(self, response, verbose=None):
         H, G = nan, nan
         M1, M2, k1, k2, phcof = nan, nan, nan, nan, nan
         headerline = []
+        centername = ''
         for idx, line in enumerate(src):
             # read in ephemerides header line; replace some field names
             if (self.query_type == 'ephemerides' and
                     "Date__(UT)__HR:MN" in line):
                 headerline = str(line).split(',')
                 headerline[2] = 'solar_presence'
-                headerline[3] = 'flags'
+                if 'Earth' in centername:
+                    headerline[3] = 'lunar_presence'
+                else:
+                    headerline[3] = 'interfering_body'
```

Comment:
Is the one-liner too long?
```suggestion
                    headerline[3] = "lunar_presence" if "Earth" in centername else "interfering_body"
```

Reply:
it was too long for what I thought the line length limit was when I was writing it -- most of the module had been wrapped at 79 -- but I have since learned that this is not a real `astroquery` restriction and that longer lines are in fact preferred. So this would work.